dev_guide/builds.adoc

```diff
@@ -659,6 +659,30 @@ code. If your application exists inside a sub-directory, you can override the
 default location (the root folder) using this field.
 ====
 
+[[max-duration]]
+
+== Build maximum duration
+When defining a `*BuildConfig*`, you can define its maximum duration, setting
+`*maxDuration*` field, it is specified in seconds and by default it is not set.
+Maximum duration is counted from Build's StartTime and it means how long a build
+pod can be active in OpenShift system (including the time needed to pull the builder
+image). After reaching specified timeout the Build is being actively terminated
```

Comment:
Is "actively" important to say here? If not:

"After reaching the specified timeout, the build is terminated by OpenShift."